apps/emqx_bridge/test/emqx_bridge_v2_testlib.erl

```diff
@@ -1943,3 +1954,19 @@ clean_aggregated_upload_work_dir() ->
     Dir = filename:join([emqx:data_dir(), bridge]),
     _ = file:del_dir_r(Dir),
     ok.
+
+%% https://stackoverflow.com/a/34883331/2708711
+all_atoms() ->
+    do_all_atoms(_N = 0, []).
+
+idx_to_atom(N) ->
+    binary_to_term(<<131, 75, N:24>>).
```

Comment:
While unlikely, this might break in the future because it is not part of documented API contract. I'd personally lean to the safer side and maybe suggest to trace `list_to_atom/1` and friends in a comment.